src/js/components/picklist/index.js

```diff
@@ -74,6 +75,10 @@ const PicklistItem = View.extend({
 
     if (this.model === this.state.get('selected')) classNames.push('is-selected');
 
+    if (this.getOption('styleForNav')) {
```

Comment:
What problem is this solving?  Ideally we solve this without modify the generic component

Reply:
Removed this in a last push. Using `itemClassName` instead.